sessions/js-fetch/star-wars/README.md

```diff
@@ -0,0 +1,27 @@
+# Fetch with Star Wars Cards
+
+Fetching data is awesome, yet it can be frightening at the beginning. But don't worry, we are going to make you familiar with the syntax and it will be fine ;)
+
+## Task
+
+Have a look at the [index.js](js/index.js) file: There is an `EXAMPLE_DATA` object with information about Luke Skywalker. We use this object to create a DOM element with the `Card()` function and append this element to the DOM with the help of `renderElement`.
+
+This approach is very static and not very handy. Lets get more data and create new cards from them dynamically!
+
+### Task 1: Fetch data from the Star Wars Api:
+
+Fetch data from the star wars api (https://swapi.dev/api/people).
```

Comment:
This API takes some time to respond. Should we mention this?

Reply:
I don't think so; by this time, students listened to a session about fetch and that it's asynchronous. Additionally, the next line tells them to use `async` and `await`, so I would say there are enough hints that this will take some time 🙂